tests/test_description.py

```diff
@@ -40,5 +40,12 @@ def test_lambda_content_with_captures():
     assert 'x == y' == e.value.description
 
 
+def test_multi_line_lambda_content_description_on_timeout():
```

Comment:
Since we're already testing the description on timeout flow. I thin it would be better to test describe function directly